### ISSUE TITLE
auro-flightline: Remove bottom padding from stops

### ISSUE THREAD

**Issue by ArcherScript** [AlaskaAirlines/auro-flightline#69]
### Is your feature request related to a problem? Please describe.

I don't feel like stops should have their bottom padding baked into the component. I think it may look nice when you see it in the documentation WITH the bottom-padding, but not every container is going to be the same. See the screenshots attached.

<img width="397" alt="image" src="https://github.com/AlaskaAirlines/auro-flightline/assets/20789620/66eaa6ca-feb1-42c3-a533-773a5516e7eb">

### Describe the solution you'd like

Remove the bottom padding from the stop's IATA code.

### Describe alternatives you've considered

Allow us to customize the CSS for stops

### Additional context

Stop's IATA code is located within the shadow dom so they are unmodifiable.

**Comment by blackfalcon** [comment]
@ArcherScript I do not believe what you are referencing is an Auro component. That is is not 100% consistent with the [Auro Flightline component seen here](https://auro.alaskaair.com/components/auro/flightline/api). 

Here is a screenshot of the flightline element, its HTML and applied styles. 

<img width="996" alt="Screenshot 2024-04-22 at 11 43 52 AM" src="https://github.com/AlaskaAirlines/auro-flightline/assets/181089/4bf78ddc-ede5-4006-b075-e3cb2ca40142">

<img width="273" alt="Screenshot 2024-04-22 at 11 46 08 AM" src="https://github.com/AlaskaAirlines/auro-flightline/assets/181089/16da69ca-002d-4ce8-b716-3be34723805f">

<img width="328" alt="Screenshot 2024-04-22 at 11 45 18 AM" src="https://github.com/AlaskaAirlines/auro-flightline/assets/181089/26a6e3af-1541-4575-a201-b733506ebb41">

I do agree that an element should NOT have any influence outside the context of the element itself, but as you can see flightline is not adding that space. 

Can you tell us more about the context of this UI so that we can assist with resolution?

**Comment by ArcherScript** [comment]
> @ArcherScript I do not believe what you are referencing is an Auro component. That is is not 100% consistent with the [Auro Flightline component seen here](https://auro.alaskaair.com/components/auro/flightline/api).
> 
> Here is a screenshot of the flightline element, its HTML and applied styles.
> 
> <img alt="Screenshot 2024-04-22 at 11 43 52 AM" width="996" src="https://private-user-images.githubusercontent.com/181089/324575234-4bf78ddc-ede5-4006-b075-e3cb2ca40142.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM4MTIzNDMsIm5iZiI6MTcxMzgxMjA0MywicGF0aCI6Ii8xODEwODkvMzI0NTc1MjM0LTRiZjc4ZGRjLWVkZTUtNDAwNi1iMDc1LWUzY2IyY2E0MDE0Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyMlQxODU0MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMDA3ZTRlM2I0ZmMzNWJjYzJlYmRkOGE5OTExMDQ2MTYzNGE3ODFmNmMwZjllMTllNjU5NmQ0ZmU1Zjg2ZjE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.57zASU0Tq-lmlh6dgX7TJbW8rhUaCmE2w3bMahAIfmU"> <img alt="Screenshot 2024-04-22 at 11 46 08 AM" width="273" src="https://private-user-images.githubusercontent.com/181089/324575862-16da69ca-002d-4ce8-b716-3be34723805f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM4MTIzNDMsIm5iZiI6MTcxMzgxMjA0MywicGF0aCI6Ii8xODEwODkvMzI0NTc1ODYyLTE2ZGE2OWNhLTAwMmQtNGNlOC1iNzE2LTNiZTM0NzIzODA1Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyMlQxODU0MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZDQ1MmE5MmQxOTg4N2JjMGZkMTRhOGYwMzZiZTk1NTBmN2VlMGJiMTQ0N2UwMTk1M2UwOTE0MzliZmJiMGFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.IXMAvMKela-KdOeELZiOC85e2MP6xRMrc2z5wCMipis"> <img alt="Screenshot 2024-04-22 at 11 45 18 AM" width="328" src="https://private-user-images.githubusercontent.com/181089/324575666-26a6e3af-1541-4575-a201-b733506ebb41.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM4MTIzNDMsIm5iZiI6MTcxMzgxMjA0MywicGF0aCI6Ii8xODEwODkvMzI0NTc1NjY2LTI2YTZlM2FmLTE1NDEtNDU3NS1hMjAxLWI3MzM1MDZlYmI0MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyMlQxODU0MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZmNlOTNhOGZlY2QwNWMyZjVkOWI0OTg4YTQ0OTllZTBhNmJiMmMzYjhmNTZiYzNkZjExODAxZmFiZGUyYzBlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.InfP0WVsqQY4EIndEW08ayyQ90HjXDSyxB6z1h2ULM8">
> I do agree that an element should NOT have any influence outside the context of the element itself, but as you can see flightline is not adding that space.
> 
> Can you tell us more about the context of this UI so that we can assist with resolution?

In the screenshot provided the stop has a `datetime` applied to it. In the screenshot below you will see bottom padding added to the IATA.

<img width="633" alt="image" src="https://github.com/AlaskaAirlines/auro-flightline/assets/20789620/645b359d-15eb-4ac5-ab18-76b5986089b1">

This screenshot is directly from the `auro-flight` documentatiokn.

Manage team has been working with the flightline and flight component and have noticed a few design issues when the component does not have dates or times applied to it. Maybe it's related to that?

**Comment by blackfalcon** [comment]
@ArcherScript this screenshot is very different from the initial one you referenced. I would suggest scheduling a meeting with an Auro engineer @jordanjones243 and our designer @michelle-alaskaair to review this issue so that we can better understand and look for a path forward. Please feel free to include your team's designer as well.

**Comment by ArcherScript** [comment]
I still would like bottom padding removed from a stop's IATA code, or at least allow us to customize styles. Manage's design team has reached out and has reached a conclusion. We have a path forward now, feel free to close this issue.
